### PR TITLE
Add a couple of Spicy-based analyzers

### PR DESCRIPTION
**Pull request by bbannier**
This PR adds a couple Spicy-based analyzers which are pulled in via zeek/spicy-analyzers.

### CODE REVIEW

File: zeek/zkg.index [hunk 1]
```diff
@@ -2,5 +2,6 @@ https://github.com/zeek/hello-world
 https://github.com/zeek/osquery-framework
 https://github.com/zeek/spicy-analyzers
 https://github.com/zeek/spicy-ldap
+https://github.com/zeek/spicy-pe
```
Comment (bbannier) [inline zeek/zkg.index:5]: There will be more packages coming, so I am moving this PR into draft state and will keep updating it.

**Comment by ckreibich** [comment]
Could you add a license to spicy-zip, Benjamin? Thanks!

Also, just so I understand — the reason these packages currently don't depend on the Spicy runtime is that `zkg` 2.12 isn't currently available out-of-the-box with released Zeek versions. Right?

**Comment by bbannier** [comment]
> Could you add a license to spicy-zip, Benjamin? Thanks!

[Done](https://github.com/zeek/spicy-zip/commit/69a5766327970956f7135e6b938ee355e42fa1db), sorry for missing that.

> Also, just so I understand — the reason these packages currently don't depend on the Spicy runtime is that `zkg` 2.12 isn't currently available out-of-the-box with released Zeek versions. Right?

Yes, correct. In principal each of the packages pulled in here should depend on zeek/spicy-plugin (creating a diamond dependency graph). We were wondering whether it would make sense to correctly model dependencies now since the latest zkg release [supports it](https://github.com/zeek/spicy-zip/commit/69a5766327970956f7135e6b938ee355e42fa1db), but decided against it since the zkg bundled with a user's Zeek install might be older. We can fix the dependencies once all supported Zeek releases come with a zkg which includes that fix.

**Comment by ckreibich** [comment]
Got it, thank you!

**Pull request merged**